utils/kata-manager.sh

```diff
@@ -777,7 +822,21 @@ handle_installation()
 	local kata_version="${7:-}"
 	local containerd_flavour="${8:-}"
 
-	[ "$only_run_test" = "true" ] && test_installation && return 0
+	local install_docker="${9:-}"
+	[ -z "$install_docker" ] && die "no install docker value"
+
+	# The tool to be testing the installation with
+	local tool="ctr"
+
+	if [ "$install_docker" = "true" ]
+	then
+		# The script provided by docker already takes care
```

Comment:
There's a small issue here: what if the user ran `-c lts -D`? The code will ignore the request to install containerd since `-D` takes priority. But that might be confusing to users so we could add a log message here. The secondary problem is that this function doesn't know how to deal with that scenario currently since `containerd_flavour` is always set.

The simplest solution might be to just document that Docker takes priority over containerd in `usage()`.

Reply:
I've added an info message.